test/data/queries.csv

```diff
@@ -0,0 +1,2 @@
+id,prompt,query
+0,summarize,"There are times when the night sky glows with bands of color. The bands may begin as cloud shapes and then spread into a great arc across the entire sky. They may fall in folds like a curtain drawn across the heavens. The lights usually grow brighter, then suddenly dim. During this time the sky glows with pale yellow, pink, green, violet, blue, and red. These lights are called the Aurora Borealis. Some people call them the Northern Lights. Scientists have been watching them for hundreds of years. They are not quite sure what causes them"
```

Comment:
Do we have to git track this? I think we can make it similar to create_dummy_csv.

Reply:
done.

Reply:
removed this.